tests/chart_tests/test_byo_sa.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from tests import supported_k8s_versions
+from tests.chart_tests.helm_template_generator import render_chart
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class TestServiceAccounts:
+    def test_SAs_created(self, kube_version):
```

Comment:
function definition should describe what is being tested, like `test_serviceaccount_rbac_disabled`